kpconv_torch/io/ply.py

```diff
@@ -1,7 +1,8 @@
+import numpy as np
 import sys
```

Comment:
```suggestion
import sys

import numpy as np
```

Split the standard library imports from the third-party library imports.